device/delta/arm64-delta_tg48m_poe-r0/delta_tg48m_poe/buffers_defaults_t1.j2

```diff
@@ -0,0 +1,45 @@
+{%- set default_cable = '300m' %}
```

Comment:
Remove this file. This not  needed as we dont support QoS for this board. contents are wrong as well